application/helpers/expressions/em_manager_helper.php

```diff
@@ -353,7 +353,8 @@ class LimeExpressionManager
      * 'hidden' => 0    // 1 if it should be always_hidden
      * 'gid' => "34"    // group id
      * 'mandatory' => 'N'   // 'Y' if mandatory, 'S' if soft mandatory
-     * 'eqn' => ""  // TODO ??
+     * 'mandSoftForced' => false // boolean True if already forced
```

Comment:
Given the sensitibity of the change, i would take the opportunity to expand on the comments. What does this 'mandSoftForced' means or is used for?

Reply:
Right ! :+1:

Reply:
> boolean value to keep Mandatroy soft question status. False if not seen, answered, not a soft mandatory or not checked one time. Check is done in self::_validateQuestion using $_POST['mandSoft']